cg/services/order_validation_service/models/errors.py

```diff
@@ -55,6 +55,11 @@ class OccupiedWellError(CaseSampleError):
     message: str = "Well is already occupied"
 
 
+class ApplicationNotValidError(SampleError):
```

Comment:
I think this error should be a `CaseSampleError` since we need both the case and sample name to uniquely identify the sample

Reply:
Fixed!